cmd/minikube/cmd/start.go

```diff
@@ -174,6 +174,23 @@ func runStart(cmd *cobra.Command, args []string) {
 		exit.WithError("Failed to generate config", err)
 	}
 
+	if viper.GetString(vmDriver) == constants.DriverNone {
+		/*
+		Caching images means:
+			- minikube download the images
+			- minikube saves them in ~/.minikube/cache
+			- minikube loads them to the host machine with `docker load -i ~/.minikube/cache/some_images`
+
+		This makes complete sense, except that when --vm-driver=none, the host machine is the local machine.
+		That means once the images is loaded into docker, it stays there even after reboots and `minikube delete`.
+		That means they are already cached and nothing needs to be done. Not only that, loading the container with
+		`docker load -i ~/.minikube/cache/some_images` is unecessary and takes time.
+		As a bonus, we save disk space.
+		*/
+		console.OutStyle("starting-none", "Caching of container images is disabled when --vm-driver=none")
```

Comment:
Remove this line - minikube, particularly with the none driver, has enough console output as is.